Trading/Mode/trading_view_signals_trading_mode/trading_view_signals_trading.py

```diff
@@ -104,9 +113,132 @@ def __init__(self, trading_mode):
         self.USE_CLOSE_TO_CURRENT_PRICE = True
         self.CLOSE_TO_CURRENT_PRICE_DEFAULT_RATIO = trading_mode.trading_config.get("close_to_current_price_difference",
                                                                                     0.02)
-        self.USE_MAXIMUM_SIZE_ORDERS = trading_mode.trading_config.get("use_maximum_size_orders", False)
+        self.BUY_WITH_MAXIMUM_SIZE_ORDERS = trading_mode.trading_config.get("use_maximum_size_orders", False)
+        self.SELL_WITH_MAXIMUM_SIZE_ORDERS = trading_mode.trading_config.get("use_maximum_size_orders", False)
         self.USE_STOP_ORDERS = False
 
+    async def create_new_orders(self, symbol, final_note, state, **kwargs):
+        data = kwargs.get("data", {})
+        user_price = float(data.get(TradingViewSignalsTradingMode.PRICE_KEY, 0))
+        user_volume = float(data.get(TradingViewSignalsTradingMode.VOLUME_KEY, 0))
+        current_order = None
+        timeout = kwargs.pop("timeout", trading_constants.ORDER_DATA_FETCHING_TIMEOUT)
+        try:
```

Comment:
If from here it is a copy of the DailyTradingMode `create_new_orders`, we should probably consider making this code generic by making it available in a library.